src/server/game/DataStores/DBCEnums.h

```diff
@@ -191,7 +191,29 @@ enum AzeriteTierUnlockSetFlags
     AZERITE_TIER_UNLOCK_SET_FLAG_DEFAULT = 0x1
 };
 
-#define BATTLE_PET_SPECIES_MAX_ID 3159
+#define BATTLE_PET_SPECIES_MAX_ID 3189
```

Comment:
Isnt this a 9.1.5 value?

Reply:
Oops, you're right. Restored the 9.1.0 value